extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/CacheInvalidate.java

```diff
@@ -18,11 +18,14 @@
  * This annotation can be combined with {@link CacheResult} annotation on a single method. Caching operations will always
  * be executed in the same order: {@link CacheInvalidateAll} first, then {@link CacheInvalidate} and finally
  * {@link CacheResult}.
+ *
+ * @deprecated Use Infinispan Cache Extension
  */
 @InterceptorBinding
 @Target({ ElementType.TYPE, ElementType.METHOD })
 @Retention(RetentionPolicy.RUNTIME)
 @Repeatable(List.class)
+@Deprecated
```

Comment:
Are these going to be removed?
If so, please add `forRemoval=true`

Reply:
yes, need to decide when. next major I guess